config/routes.rb

```diff
@@ -1639,6 +1639,7 @@
         alert_profile_edit
         alert_profile_field_changed
         button
+        clear_search_text
```

Comment:
@GregP i understand that but the change https://github.com/ManageIQ/manageiq/pull/10453/files/55f09e87aa9b713cc1681ec8b29966f386dc6e3b#diff-23ab9acb538585d8324a9f4d1497e5a5R1608 makes it generic to all controllers so this needs to be moved into a common routes block that is included by all controllers that use advanced search.